release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,90 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
```

Comment:
Is it standard to simply say `this release` or is it standard to say `for the 4.11 release`?

Reply:
I think it's fine to leave as-is, so I'm not changing it.